src/amitaRJun20/CheckCharacterMethods.java

```diff
@@ -0,0 +1,38 @@
+package amitaRJun20;
+
+public class CheckCharacterMethods {
+	
+	void testCharMethod(String name,char ch) {
+		int digitCount=0;
+		int upperCaseCount=0;
+		int lowerCaseCount=0;
+		int specialCharCount=0;
+		
+		for(int index=0;index<name.length();index++) {
+			if(Character.isDigit(name.charAt(index))) {
+				digitCount++;
+			}
+			else if(Character.isLetter(name.charAt(index))) {
```

Comment:
Here else if(Character.isLetter(name.charAt(index))) you don't need this line beacuse letters will already going to cover under uppercase and lowercase. Just remove this line and write an individual else if block for uppercase and lower case.